docs/reduction-in-locks.md

```diff
@@ -40,7 +40,7 @@ The `--lock-ddl=REDUCED` option features are as follows:
             | 400 GB                     | 3888 sec         | 3.948 sec             | 984.80                   |
             | 500 GB                     | 4896 sec         | 4.065 sec             | 1204.43                  |
         
-        Using `--lock-ddl=REDUCED` leads to a dramatic reduction in backup time compared to `--lock-ddl=ON`, especially with larger data sizes.
+        Using `--lock-ddl=REDUCED` leads to a dramatic reduction in backup lock time compared to `--lock-ddl=ON`, especially with larger data sizes.
```

Comment:
Would this be more clear?

Using --lock-ddl=REDUCED significantly reduces the time DDL operations are blocked during backup, although the total backup time remains unchanged. The data size particularly impacts the lock duration - larger databases benefit more from this reduced locking approach.